README.md

```diff
@@ -95,7 +95,8 @@ Assuming the app is bound to a Conjur service instance, pushing the app will res
 
```

Comment:
The README should also include notes on how to update summon and conjur-api-go for developers - right now they are pinned to specific releases, but we may want to update that in the future and it would be good to have notes on how to do that.